samples/guestbook/aws/provision_db/main.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The provision_db program connects to an RDS database and initializes it with
+// SQL from stdin. It is intended to be invoked from Terraform.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func main() {
+	log.SetFlags(0)
+	log.SetPrefix("aws/provision_db: ")
+	flags := map[string]*string{
```

Comment:
(Here and elsewhere:) Using a map loses the compile safety of passing these variables directly. Use variables directly or perhaps a struct to group them.

Reply:
Done.